lib/BookmarkPanelHub.jsm

```diff
@@ -64,6 +64,10 @@ class _BookmarkPanelHub {
    * @returns {obj|null} response object or null if no messages matched
    */
   async messageRequest(target, win) {
+    if (!this._initalized) {
```

Comment:
This is actually misspelled, can you change this (and other usages) to`_initialized`